tests/store/test_store_api.py

```diff
@@ -30,13 +34,16 @@ def test_fetch_past_files(populated_store, bundle_data_old, timestamp, old_times
     # WHEN fetching all files before the oldest date
     date = old_timestamp + datetime.timedelta(days=10)
     assert old_timestamp < date < timestamp
-    query = store.files_before(before=str(date))
+    files: List[File] = store.get_files_before(before_date=date)
+
+    # THEN a list of Files is returned
+    assert isinstance(files[0], File)
 
     # THEN assert only files from the old bundle was found
-    assert len(query.all()) == 2
+    assert len(files) == 2
 
 
-def test_fetch_no_files_before_oldest(populated_store, bundle_data_old, old_timestamp, timestamp):
+def test_fetch_no_get_files_before_oldest(populated_store, bundle_data_old, old_timestamp, timestamp):
```

Comment:
Typehints and suggestions
```suggestion
def test_get_no_get_files_before_oldest(populated_store, bundle_data_old, old_timestamp, timestamp):
```